nginx/assets/monitors/5xx.json

```diff
@@ -0,0 +1,32 @@
+{
+	"name": "[NGINX] 5xx Errors",
+	"type": "query alert",
+	"query": "avg(last_4h):anomalies(avg:nginx.upstream.peers.responses.5xx{*}.as_count(), 'basic', 2, direction='above', alert_window='last_15m', interval=60, count_default_zero='true') >= 1",
```

Comment:
Same timeframe comment as for 4xx monitor.

Reply:
same comment regarding the space aggregator.